resources/views/contribution.blade.php

```diff
@@ -1,4 +1,68 @@
 @extends('layouts.app')
 
+@section('title', 'Contribution')
 @section('content')
-<h1>This is contribution Page</h1>
\ No newline at end of file
+    <div class="row">
+        <div class="col-sm-12">
+            <form method="POST" action="/contribution/add">
+                <div class="form-group">
+                    {{ csrf_field() }}
+                    <p class="ext-monospace">Title</p>
+                    <input type="text" name="title" class="form-control @if ($errors->has('title')) is-invalid @endif" 
```

Comment:
You can use ternary operator here instead of @if @endif, since it's just online. We discussed this before, but here is a reference: https://stackoverflow.com/questions/25284344/ternary-in-laravel-blade